uber/tests/conftest.py

```diff
@@ -1,24 +1,49 @@
 from uber.common import *
```

Comment:
That makes sense, but sometimes people prefer the word-like capitalization, e.g. Perl instead of PERL, so I figured it was worth asking.